src/components/diagrams/diagram-header.js

```diff
@@ -14,6 +15,10 @@ import PushPinOutlinedIcon from '@mui/icons-material/PushPinOutlined';
 import CloseIcon from '@mui/icons-material/Close';
 import PropTypes from 'prop-types';
 import makeStyles from '@mui/styles/makeStyles';
+import clsx from 'clsx';
+import { stopDiagramBlink } from '../../redux/actions';
+
+const BLINK_LENGTH_MS = 3000;
```

Comment:
It's slightly slow for me, maybe 2 sec with '0%, 20%' ? I don't know what think the PO about it, it's their choice at the end

Reply:
Ok, I slightly changed it, now 1800ms and 0-25%